invenio_administration/assets/semantic-ui/js/invenio_administration/search/SearchResultItem.js

```diff
@@ -12,13 +12,25 @@ import { Table } from "semantic-ui-react";
 import isEmpty from "lodash/isEmpty";
 import { Actions } from "../actions/Actions";
 import { withState } from "react-searchkit";
+import DateFormatter from "../components/DateFormatter";
 
 class SearchResultItemComponent extends Component {
   refreshAfterAction = () => {
     const { updateQueryState, currentQueryState } = this.props;
     updateQueryState(currentQueryState);
   };
 
+  displayValue = (property) => {
+    const { resourceSchema, result } = this.props;
+    const type = _get(resourceSchema.properties[property], "type");
+    const value = _get(result, property);
+    if (type === "date") {
```

Comment:
what if we add another type ? there will be a lot of conditional statements. Any other way of applying the formatter to avoid the ifs ??